src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,7 +1,205 @@
+            </HeaderVideo>
+          </Header>
+          <Content>
+            <ContentsTop>
+              <PosterWrap>
+                <Poster src={musicalData.images?.[0]} alt="포스터" />
+              </PosterWrap>
+              <PlayTitle>
+                <Grade grade={musicalData.grade}>{musicalData.grade}</Grade>
+                <p>{musicalData.title}</p>
+              </PlayTitle>
+              <Rating>
+                <span>예매율</span>
+                <p>{musicalData.rating}%</p>
+              </Rating>
+              <PlayInfo>
+                <li>
+                  <em>시간정보</em>
+                  <span>{musicalData.period} ㅣ </span>
+                  <span>{musicalData.time}분</span>
+                </li>
+                <li>
+                  <em>극장</em>
+                  <span>{musicalData.theater}</span>
+                </li>
+                <li>
+                  <em>출연진</em>
+                  {musicalData.casting?.map(actor => {
+                    return <span key={actor.id}>{actor} ㅣ </span>;
+                  })}
+                </li>
+              </PlayInfo>
+              <TicketBtn>예매하기</TicketBtn>
+            </ContentsTop>
+            <ContentBody musicalData={musicalData} />
+          </Content>
+        </>
+      )}
+    </div>
+  );
 }
 
+const HeaderVideo = styled.div`
+  width: 100%;
+  z-index: -1;
+  height: 570px;
+  background-color: #000000;
+
+  img {
+    width: 100%;
+    height: 100%;
+    background-size: contain;
+  }
+
+  div {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: #000000;
+    opacity: 30%;
+  }
+`;
+
+const Header = styled.div`
+  position: relative;
+  display: flex;
+`;
+
+const Content = styled.div`
+  width: 980px;
+  margin: 0 auto;
+  padding: 20px 0 0 0;
+`;
+
+const PosterWrap = styled.div`
+  position: absolute;
+  top: -40px;
+  left: 0;
+  width: 205px;
+  height: 305px;
+  z-index: 1;
+`;
+
+const Poster = styled.img`
+  width: 100%;
+  border-radius: 4px;
+`;
+
+const TicketBtn = styled.button`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: 17%;
+  background: linear-gradient(to right, #6200ee, #7c21ff);
+  border-radius: 24px;
+  outline: 0;
+  border: 0;
+  padding: 16px 24px;
+  font-size: 16px;
+  color: #ffffff;
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: 0 0 5px 3px rgba(124, 33, 255, 0.2);
+    transition: all 0.2s;
+  }
+`;
+
+const ContentsTop = styled.div`
+  position: relative;
+  padding-left: 245px;
+  margin-bottom: 50px;
+  border-bottom: 1px solid transparent;
+`;
+
+const PlayTitle = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+
+  p {
+    font-size: 32px;
+    font-weight: bold;
+    margin-left: 16px;
+  }
+`;
+
+const Grade = styled.div`
+  background-color: ${({ grade }) =>
+    grade === 18
+      ? '#FF0363'
+      : grade === 15
+      ? '#F48131'
+      : grade === 12
+      ? '#F5C951'
+      : '#36C640'};
```

Comment:
```suggestion
  background-color: ${({ grade }) =>{
  const gradeScore = {
  18:'#FF0363',
  15:'#F48131'
  12:'#F5C951'
  }
  return gradScore[grade]||'#36C640'
  }
```
조건문으로 분기처리 할 수 있지만,가독성 측면에서 객체로 리팩토링 할 수 있습니다.
객체 + computedProperty 활용해보시면 됩니다!
|| 연산자의 단축평가는 좌항이 null이나 undefined라면 우항을 반환합니다.
전달하는 key가 객체에 없다면 default로 '#36C640'이 값을 반환하게 만드는 코드 입니다.